packages/notification-center-angular/package.json

```diff
@@ -31,30 +31,30 @@
     }
   },
   "dependencies": {
-    "@angular/common": "^15.0.0",
-    "@angular/compiler": "^15.0.0",
-    "@angular/core": "^15.0.0",
-    "@angular/platform-browser": "^15.0.0",
-    "@angular/platform-browser-dynamic": "^15.0.0",
+    "@angular/common": "~15.1.0",
+    "@angular/compiler": "~15.1.0",
+    "@angular/core": "~15.1.0",
+    "@angular/platform-browser": "~15.1.0",
+    "@angular/platform-browser-dynamic": "~15.1.0",
```

Comment:
With version `15.0.0` had compatibility problems with the TS version 4.9.5. Went for the upgrade and the tilde modifier to allow only patch versions.